app/scss/components/_grid.scss

```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```

Comment:
@kisphp Yes we aren't using Bootstrap because bootstrap is unnecessary bloat and that's why there was styleguides.html (Can be seen at public/styleguides.html) to show everyone our own components and how they can be used